ledger/sandbox/src/main/scala/com/digitalasset/platform/index/LedgerBackedIndexService.scala

```diff
@@ -270,6 +270,15 @@ abstract class LedgerBackedIndexService(
           .getOrElse(Future.successful(None)))
       .flatMapConcat(ledger.configurationEntries(_).map(_._2.toDomain))
 
+  override def getLedgerConfiguration(): Source[LedgerConfiguration, NotUsed] =
```

Comment:
May be out of scope, but should this also take a `startExclusive` parameter?

Reply:
No, but the implementation should emit the latest configuration and then publish updates. But honestly, configurations change very seldom, so this shouldn't really be a streaming response.

Reply:
I agree with that last sentence.